lib/clover_field.cpp

```diff
@@ -244,7 +244,7 @@ namespace quda {
       param.reconstruct = false; // we cannot use a compressed field for storing the inverse
       CloverField clover_inverse(param);
       clover_inverse.copy(src, false);
-      cloverInvert(clover_inverse, true);
+      cloverInvert(clover_inverse, false);
```

Comment:
Why change to false? Just b/c you don't want the trace of the inverse?

Reply:
Good eyes 😄 I have changed how this kernel works: this boolean previously was used to dictate whether to compute the trace log of the clover field or not.  I have changed this option, to now dictate whether we will only compute the trace log (`true`) or to also compute the inverse field.  This is documented in the doxygen,

This allows us to check the trace log is well defined, which is a proxy for checking that the clover matrix is invertible.  There was previously a hole, where we would fail to catch failed Cholesky if doing dynamic clover, since the clover inverse kernel wasn't called.  We now always call it as a sanity check, but only optionally store the inverse field.